### PR TITLE
protractor option disable controlflow option in e2e script

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
option to disable controlflow in e2e script

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/proxy.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@
 import { browser } from 'protractor';
 
 export async function setConfigField(field: string, value: string) {
+
     return browser.executeScript(
-        `window.adf.setConfigField('${field}', '${value}');`
+        "window.adf.setConfigField(`"+field + "`, `" + value + "`);"
```
Comment (DenysVuika) [inline e2e/proxy.ts]: why replacing ES6 feature with ES5 code?

Comment (eromano) [reply e2e/proxy.ts]: yep is a problem when you put in it a string with the ' ' single quote

Comment (eromano) [reply e2e/proxy.ts]: you can try wit for example the search configuration

Comment (DenysVuika) [reply e2e/proxy.ts]: I see, makes sense

**Pull request merged**